x-pack/legacy/plugins/siem/public/components/page/network/network_http_table/translations.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const HTTP_REQUESTS = i18n.translate('xpack.siem.networkHttpTable.title', {
+  defaultMessage: 'HTTP Requests',
+});
+
+export const UNIT = (totalCount: number) =>
+  i18n.translate('xpack.siem.networkHttpTable.unit', {
+    values: { totalCount },
+    defaultMessage: `{totalCount, plural, =1 {request} other {requests}}`,
+  });
+
+export const METHOD = i18n.translate('xpack.siem.networkHttpTable.column.method', {
+  defaultMessage: 'Method',
+});
+export const DOMAIN = i18n.translate('xpack.siem.networkHttpTable.column.domain', {
+  defaultMessage: 'Domain',
+});
+
+export const PATH = i18n.translate('xpack.siem.networkHttpTable.column.path', {
+  defaultMessage: 'Path',
+});
+
+export const STATUS = i18n.translate('xpack.siem.networkHttpTable.column.status', {
+  defaultMessage: 'Status',
+});
+
+export const LAST_HOST = i18n.translate('xpack.siem.networkHttpTable.column.status', {
```

Comment:
```suggestion
export const LAST_HOST = i18n.translate('xpack.siem.networkHttpTable.column.lastHostTitle', {
```

i18n keys need a little cleanup in this file -- there's a few dupes and missing [message types](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types). I think you can use `Title` for most these (since they're columns...at least that's what we did elsewhere :).